python/sbp/client/drivers/pyserial_driver.py

```diff
@@ -45,9 +45,10 @@ def __init__(self, port, baud=115200, rtscts=False):
     try:
       handle = serial.serial_for_url(port)
       handle.baudrate = baud
-      handle.timeout = 1
+      handle.timeout = 0.03
```

Comment:
Can we add the timeout (in seconds) and the read chunk size (in bytes)  as arguments to the constructor?